Plugins/NativePerformanceLogger/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(SOURCES
+    "Include/Babylon/Plugins/NativePerformanceLogger.h"
+    "Source/NativePerformanceLogger.cpp")
+
+add_library(NativePerformanceLogger ${SOURCES})
```

Comment:
Maybe it would be better to name this new plugin `NativeTracing`. Any opinions on this?